regression/scripts/B737/mission_B737.py

```diff
@@ -84,7 +84,7 @@ def main():
     old_results = load_results()   
 
     # plt the old results
-    plot_mission(results)
+    plot_mission_new(results)
```

Comment:
what is the purpose of this?

Reply:
Are we planning to keep both of these around?

Reply:
I fixed this. Since I aliased conditions back it all works.